src/rust/scheduler/waker64.rs

```diff
@@ -15,7 +15,7 @@ use ::std::{
 //==============================================================================
 
 /// Log2 of [WAKER_BIT_LENGTH].
-pub const WAKER_BIT_LENGTH_SHIFT: usize = 6;
+pub const WAKER_BIT_LENGTH_SHIFT: usize = 5;
 
 /// Number of Bits in a [Waker64]
 pub const WAKER_BIT_LENGTH: usize = 1 << WAKER_BIT_LENGTH_SHIFT;
```

Comment:
This changes more than it appears.

Reply:
Lol, this is the most mysterious comment that I have ever seen.

Reply:
I just wanted to point out that this is the place where we actually change the length of `WAKER`s.